core/internal/gethwrappers2/compile.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
ported from ccip as is, might be a way to merge with gethwrappers1 as a follow up